origin-discovery/src/listener/utils.js

```diff
@@ -112,7 +113,13 @@ async function withRetrys(fn, exitOnError = true) {
   }
 }
 
+function bytes32ToIpfsHash(hashHex) {
```

Comment:
Maybe we should have an origin-util package where we put this kind of common utility method ?
I'm not suggesting we do this in this PR, just thinking in the future...

Reply:
Yea, agreed, we need to do that we have the reverse of this function floating around in a few places as well.